src/Databases/DatabaseAtomic.cpp

```diff
@@ -53,16 +53,23 @@ DatabaseAtomic::DatabaseAtomic(String name_, String metadata_path_, UUID uuid, c
     , db_uuid(uuid)
 {
     assert(db_uuid != UUIDHelpers::Nil);
-    fs::create_directories(fs::path(getContext()->getPath()) / "metadata");
-    fs::create_directories(path_to_table_symlinks);
-    tryCreateMetadataSymlink();
```

Comment:
Why can't we create directories in the constructor now?

Reply:
clickhouse-local is used most of the time without using the database, and creating and removing a lot of directories slows it down.